src/ui/AdvancedSearch/AdvancedSearch.ts

```diff
@@ -34,29 +35,46 @@ export class AdvancedSearch extends Component {
    * @componentOptions
    */
   static options: IAdvancedSearchOptions = {
+
     /**
-     * Specifies whether or not to include the built-in keywords section.
-     * Default is `true`
+     * Specifies whether to include the built-in **Keywords** section.
+     *
+     * Default value is `true`.
      */
     includeKeywords: ComponentOptions.buildBooleanOption({ defaultValue: true }),
 
     /**
-     * Specifies whether or not to include the built-in date section.
-     * Default is `true`
+     * Specifies whether to include the built-in **Date** section.
+     *
+     * Default value is `true`.
      */
     includeDate: ComponentOptions.buildBooleanOption({ defaultValue: true }),
 
     /**
-     * Specifies whether or not to include the built-in document section.
-     * Default is `true`
+     * Specifies whether to include the built-in **Document** section.
+     *
+     * Default value is `true`.
      */
     includeDocument: ComponentOptions.buildBooleanOption({ defaultValue: true })
   };
 
+  /**
+   * The list of all available inputs inside the AdvancedSearch component.
```

Comment:
Take care when adding JSDOC comments to stuff that is not already documented by us.

Everything that becomes part of the reference documentation means we cannot ever break it (because we document it publicly), and that we need to ensure we have proper testing around the "non-breaking" part of that piece of code.

In this case, someone using the framework should not have to know or use this variable. So, we should remove it.